src/IMirrorNodeResponses.sol

```diff
@@ -39,4 +39,14 @@ interface IMirrorNodeResponses {
         int64 amount;
         string denominating_token_id;
     }
+
+    struct TokenRelationship {
+        bool automatic_association;
+        uint256 balance;
+        string created_timestamp;
+        uint256 decimals;
+        string token_id;
```

Comment:
to be able to use it with `parseJson`/`decode`, does this need to be sorted alphabetically?

Reply:
I think it is indifferent to the order, but I will need to confirm

Reply:
I copied the same order that we get from the response by the mirror node, I think decode is indifferent to the order, but I will need to confirm.